packages/utils/package.json

```diff
@@ -112,28 +112,26 @@
   "dependencies": {
     "@chainsafe/is-ip": "^2.0.2",
     "@libp2p/interface": "^1.1.0",
-    "@libp2p/peer-collections": "^5.1.2",
```

Comment:
Why was this dependency removed? It's used in the [peer-job-queue](https://github.com/libp2p/js-libp2p/blob/main/packages/utils/src/peer-job-queue.ts#L4)

Reply:
It was removed in #2339, this PR depended on that one, I thought I'd mentioned that in the description but I guess I missed it..